server/store.go

```diff
@@ -123,6 +135,7 @@ func (s *storeInfo) getLocationID(keys []string) string {
 type StoreStatus struct {
 	*pdpb.StoreStats
 
+	Blocked           bool      `json:"blocked"`
```

Comment:
blocked may be confuse user? what is blocked? maybe we should think a better name.